src/App.jsx

```diff
@@ -1,8 +1,4 @@
-import React, {Component} from 'react';
+import React from 'react';
 
-class App extends Component {
-  render() {
-    return (<h1>Welcome to Pubcrawler</h1>);
-  }
-}
+const App = () => <h1>Welcome to pubcrawler</h1>;
```

Comment:
Well, the linter complained, so I turned it into a stateless function. I expect this to change pretty soon.